core/src/main/scala/org/apache/spark/util/collection/ExternalSorter.scala

```diff
@@ -113,11 +114,21 @@ private[spark] class ExternalSorter[K, V, C](
     if (shouldPartition) partitioner.get.getPartition(key) else 0
   }
 
+  private val metaInitialRecords = conf.getInt("spark.shuffle.sort.metaInitialRecords", 256)
+  private val kvChunkSize = conf.getInt("spark.shuffle.sort.kvChunkSize", 8192)
```

Comment:
Also, any reason to make this tunable in the first place? I think ideally we'd just have a value we're comfortable hard coding.